arch/arm/core/fault.c

```diff
@@ -350,6 +356,33 @@ static void _BusFault(const NANO_ESF *esf, int fromHardFault)
 	}
 #endif /* !defined(CONFIG_ARMV7_M_ARMV8_M_FP) */
 
+#if defined(CONFIG_NXP_MPU)
+	u32_t sperr = SYSMPU->CESR & SYSMPU_CESR_SPERR_MASK;
+	u32_t mask = BIT(31);
+	int i;
+
+	if (sperr) {
+		PR_EXC("  NXP MPU error\n");
+		for (i = 0; i < SYSMPU_EAR_COUNT; i++, mask >>= 1) {
+			if (!(sperr & mask)) {
+				continue;
+			}
+			STORE_xFAR(edr, SYSMPU->SP[i].EDR);
+			STORE_xFAR(ear, SYSMPU->SP[i].EAR);
+
+			PR_EXC("    Port: %x, %s mode, %s Address: 0x%x, "
```

Comment:
all comes out in one line, can this be put in multiple lines, depending on the serial console client used this might garble the screen.

```
DEVICE:   NXP MPU error
DEVICE:     Port: 3, Supervisor mode, Instruction Address: 0x20000d8c, Type: Read, Master: 0, Regions: 0x8210
```

Reply:
Should Port & Master be %d instead of %x?  If %x than we should probably have "0x" so its clear.